src/domElements.js

```diff
@@ -0,0 +1,138 @@
+// Thanks to ReactDOMFactories for this handy list!
+
+export default [
```

Comment:
feature detection consistency maintained by tests. there are some tests by attributes, we need also add tests by features